en/sql/transaction.rst

```diff
@@ -802,6 +802,16 @@ CUBRID determines the lock mode depending on the type of operation to be perform
     | requesting SCH_M_LOCK but waiting T2's SCH_S_LOCK is released |                                                               |
     +---------------------------------------------------------------+---------------------------------------------------------------+
 
+*   **Special Locks**
+
+    **CUBRID 10.2** introduces a new type of locks, which are used in a few special cases of internal operations.
+
+    *   **Bulk Update Lock, BU_LOCK**
+
+        This lock is used whenever we want to insert multiple records at a time into the database, **in parallel**. Since multiple records are inserted at the same time, instance locking in this case would prove to be inefficient, so we use a strong lock on the class instead.
```

Comment:
> This lock is used whenever we want to insert multiple records at a time into the database, **in parallel**.
This may mislead users. It is better to clearly mention that "loaddb" pf CS mode uses it.

Reply:
The information required for BU_LOCK

- Designed for the purpose of bulk inserting large amount of data
- When transaction has BU_LOCK on class, it won't need instance locks
- BU_LOCK is compatible with itself and SCH_S_LOCK. It is not compatible with other locks
- BU_LOCK is currently used only by loaddb client-server sessions.